code/game/machinery/portable_turret.dm

```diff
@@ -547,7 +547,7 @@ var/list/turret_icons
 		assess_and_assign(SP.pilot, targets, secondarytargets)
 
 	for(var/obj/vehicle/T in view(scan_range,src))
-		assess_and_assign(T.buckled_mob, targets, secondarytargets)
+		assess_and_assign(T.buckled_mobs[1], targets, secondarytargets)

```

Comment:
First item in the list is [1], not [0]?

Reply:
i will touble check TG but i think this is proper...time to break out some mindless mobs later.